DCO

```diff
@@ -1,7 +1,7 @@
 Developer Certificate of Origin
 Version 1.1
 
-Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
+Copyright (C) 2004, 20019 The Linux Foundation and its contributors.
```

Comment:
This should still stay as 2006. Copyright years only get updated if the material itself is updated.